app/changes/ana_move-dependabot-config-to-root-folder

```diff
@@ -0,0 +1 @@
+[Code Improvements] [#4216](https://github.com/cosmos/lunie/pull/4216) Move dependabot config to root folder of monorepo @Bitcoinera
```

Comment:
why did you copy this in app as well?

Reply:
I see.

> We should have some kind of global changelog 🤔 ? Idk, adding changelogs for all subrepos for now

yep, will think about an update to simsala

Reply:
Great stuff! 🤩 Simsala will now be monorepo-compatible

Reply:
![egleniyormuyuzcom-cilBKvPHOrWHO7YCAs](https://media.giphy.com/media/EU4y1ENWrcZA4/giphy.gif)